src/main/resources/application.properties

```diff
@@ -7,17 +7,38 @@
 #zeebe.client.cloud.clientSecret=XXX
 
 # or local
-#zeebe.client.broker.gateway-address=127.0.0.1:26500
-#zeebe.client.security.plaintext=true
+#camunda.client.zeebe.base-url=http://localhost:26500
+#camunda.client.zeebe.gateway-url=http://localhost:26500
 
+camunda.client.mode=oidc
+camunda.client.auth.client-id=zeebe
+camunda.client.auth.client-secret=tSpZfhCadVrSXqIpMtdMKveczmqFXnkD
```

Comment:
is this a real secret? can we avoid putting it and also the issuer into public code in any case?

Reply:
updated to use env var